ledger/participant-state-metrics/src/main/scala/com/daml/ledger/participant/state/v2/metrics/TimedWriteService.scala

```diff
@@ -81,4 +81,6 @@ final class TimedWriteService(delegate: WriteService, metrics: Metrics) extends
 
   override def currentHealth(): HealthStatus =
     delegate.currentHealth()
+
+  override def isDeduplicationEnabled: Boolean = false
```

Comment:
This needs to call `delegate.isDeduplicationEnabled`.

Reply:
🤦